docs/embedding/sdk/upgrade.md

```diff
@@ -10,8 +10,8 @@ Here's a basic overview of the steps you'll want to take when upgrading your SDK
 ## 1. Read the release post and changelog for both Metabase and the Embedded analytics SDK
 
 - [Release posts](https://www.metabase.com/releases) give a good overview of what's in each release, and call out breaking changes (which are rare).
-- [Metabase changelogs](https://www.metabase.com/changelog) list Metabase changes.
-- [Embedded analytics SDK changelogs](https://www.metabase.com/changelog/55) list changes specific to the SDK package.
+- [Metabase + Embedding analytics SDK changelogs](https://www.metabase.com/changelog) list Metabase and Embedding analytics SDK changes.
+- [Embedded analytics SDK package changelog](https://github.com/metabase/metabase/blob/release-x.57.x/enterprise/frontend/src/embedding-sdk-package/CHANGELOG.md) list changes specific to the SDK lightweight `@metabase/embedding-sdk-react` package.
```

Comment:
It points to 57 (before 57 is cut), to guarantee that we have a proper `embedding-sdk-package` folder in url.